components/browser/toolbar/src/main/java/mozilla/components/browser/toolbar/display/DisplayToolbar.kt

```diff
@@ -177,7 +177,7 @@ class DisplayToolbar internal constructor(
      */
     var colors: Colors = Colors(
         securityIconSecure = ContextCompat.getColor(context, R.color.photonWhite),
-        securityIconInsecure = ContextCompat.getColor(context, R.color.photonWhite),
+        securityIconInsecure = Color.TRANSPARENT,
```

Comment:
Would be nice to know what was the initial reason for having all these defaults set to white, whether there is any functionality depending on this.
Setting transparent here will force clearing color filters in updateSiteSecurityIcon thus fixing the buggy scenario seen on Focus but I understand this was tested on Focus and Fenix with no visible ill effects so this should be safe to try.
For consistency could we also update to transparent the default for `securityIconSource`?